stable/concourse/values.yaml

```diff
@@ -140,6 +140,12 @@ concourse:
       ##
       # region:
 
+      ## Configure authentication using an access key and secret key. If disabled, IAM role auth is assumed.
+      ## Session Token can also be enabled, if required.
+      keyAuth:
+        enabled: true
+        # useSessionToken: false
```

Comment:
Would you mind leaving this uncommented with `false` there? The reason why I ask that is that with https://github.com/helm/charts/pull/11296 the idea is to move forward to a place where we can automate the verification that every possible value is documented (requiring variables to not be commented then).

Thx!

Reply:
Done.